linux/system/repo.sls

```diff
@@ -58,6 +58,9 @@ linux_repo_{{ name }}:
   {%- if repo.key_url is defined %}
   - key_url: {{ repo.key_url }}
   {%- endif %}
+  - consolidate: {{ repo.get('consolidate', False) }}
+  - clean_file: {{ repo.get('clean_file', False) }}
```

Comment:
It should be default to set this to true by default as currently we don't allow multiple repositories in one sources.list.d file to be managed by salt.